src/planscape/config/treatment_goals.json

```diff
@@ -1,20 +1,106 @@
-             "priorities": [
+              "short_question_text": "High-severity Fire Areas",
+              "scenario_priorities": [
                 "probability_of_fire_severity_high"
-             ],
-             "weights": [
-                5
-             ]
+              ],
+              "weights": [],
+              "scenario_output_fields": [
+                "high_severity_potential_smoke_emissions",
+                "total_fuel_exposed_to_fire",
+                "dead_and_down_carbon",
+                "structure_exposure",
+                "damage_potential_wui",
+                "standing_dead_and_ladder_fuels",
+                "available_standing_biomass",
+                "sawtimber_biomass",
+                "costs_of_potential_treatment_moving_biomass",
+                "costs_of_potential_treatment_moving_sawlogs",
+                "california_spotted_owl"
+              ],
+              "stand_thresholds": [],
+              "global_thresholds": []
+            },
+            {
+              "long_question_text": "Where are the best opportunities to effectively reduce fire risk to the Wildland Urban Interface?",
+              "short_question_text": "Reduce WUI Fire Risk",
+              "scenario_priorities": [
+                "structure_exposure_score",
+                "wui_damage_potential"
+              ],
+              "weights": [],
+              "scenario_output_fields": [
+                "high_severity_potential_smoke_emissions",
+                "moderate_severity_potential_smoke_emissions",
+                "total_fuel_exposed_to_fire",
+                "dead_and_down_carbon",
+                "standing_dead_and_ladder_fuels",
+                "available_standing_biomass",
+                "sawtimber_biomass",
+                "heavy_fuel_load",
+                "california_spotted_owl"
+              ],
+              "stand_thresholds": [],
+              "global_thresholds": []
+            },
+            {
+              "long_question_text": "Where are the optimal areas to treat to reduce overall fuel loads?",
+              "short_question_text": "Fuel-load Reduction Opportunities",
+              "scenario_priorities": [
+                "total_fuel_exposed_to_fire"
+              ],
+              "weights": [],
+              "scenario_output_fields": [
+                "high_severity_potential_smoke_emissions",
+                "moderate_severity_potential_smoke_emissions",
+                "dead_and_down_carbon",
+                "standing_dead_and_ladder_fuels",
+                "available_standing_biomass",
+                "sawtimber_biomass",
+                "heavy_fuel_load",
+                "probability_of_fire_severity_high",
+                "probability_of_fire_severity_moderate",
+                "california_spotted_owl"
+              ],
+              "stand_thresholds": [
+                "Probability of Fire Severity (High) > 50% TODO Replace with actual threshold string"
```

Comment:
Was waiting on these strings to make this pr, but decided to at least get the structure out so we can update these later. These should be made available on the same doc that the rest of the treatment goals are documented on (hopefully soon)

Reply:
Updated with threshold strings